framework/src/actions/AddFunctionAction.C

```diff
@@ -29,5 +29,9 @@ AddFunctionAction::AddFunctionAction(InputParameters params) :
 void
 AddFunctionAction::act()
 {
+  FunctionParserBase<Real> fp;
+  std::string vars = "x,y,z,t,NaN,pi,e";
```

Comment:
Yes, that is where I grabbed it. It would be nice if this sort of thing lived in a common place.